hack/generate_release_notes.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# run from repository root
+
+"""
+Скрипт для генерации файлов RELEASE_NOTES.md и RELEASE_NOTES.ru.md
+из файлов в папке CHANGELOG.
+
+Скрипт парсит YAML файлы с изменениями и создает markdown файлы
+с релизными заметками в требуемом формате.
+"""
```

Comment:
Not sure about having comments in Russian here. Maybe keep code comments in English as we have in other files?